src/ui/public/styles/pagination.less

```diff
@@ -42,6 +42,7 @@ paginate {
             text-decoration: none !important;
             font-weight: bold;
             color: @paginate-page-link-active-color;
+            cursor: default;
```

Comment:
Whoops, you're right man. didn't realize/forgot [default sets it to a regular cursor](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor).